src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -26,6 +26,21 @@ namespace Microsoft.PowerShell.EditorServices.Protocol.Server
 {
     public class LanguageServer
     {
+        private const string CheckHelpScript = @"
+[CmdletBinding()]
+param (
+    [String]$CommandName
+)
+Try {
```

Comment:
Below you use a lower case `try` (and everywhere else uses lower case keywords).  Probably should tweak that here to be consistent.  :-)

Reply:
I'll get this corrected after the below comment is replied to so I'm not doing a bunch of extra work :)